db/migrate/20220302162344_create_slug_tags.rb

```diff
@@ -0,0 +1,15 @@
+class CreateSlugTags < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def up
+    Exercise.preload(:publication).in_batches(of: 100, load: true) do |exercises|
+      Exercise.transaction do
+        exercises.each(&:save) # creates the slug tags but does not set updated_at
```

Comment:
Wouldn't you need `touch: false` here to not set `updated_at`?

Reply:
Because there's no change in the exercise record, the exercises DB row is not updated... the hooks create the associated records but these don't set updated_at on the exercise row.

Reply:
Ah I saw the tags assignment and assumed it'd mark the record as dirty, but reading the tag implementation clarifies it for me.